### PR TITLE
commit ass6

### PR DESCRIPTION
**Pull request by AbigailZaw**
May I know if I have something to fix sir.

### CODE REVIEW

File: yt_studio.html [hunk 1]
```diff
@@ -36,11 +37,70 @@
...
+                                <div class="cmtProfileContainer">
+                                    <img src="img/img3.jpg" class="cmtProfile"/>
+                                </div>
+                                <div class="name fs op">
+                                    Ko Soe Htet Aung . 2 day ago
+                                </div>
+                                <div class="cmtImageContainer">
+                                    <img src="img/sddefault.jpg" class="cmtImage"/>
+                                </div>
+                            </div>
```
Comment (james-highground) [inline yt_studio.html]: indentation here should match opening tag

File: yt_studio.html [hunk 2]
```diff
@@ -36,11 +37,70 @@
...
+                                <div class="fs commentLetter">
```
Comment (james-highground) [inline yt_studio.html]: this indentation should be moved in

File: yt_studio.html [hunk 3]
```diff
@@ -56,8 +116,39 @@
...
+                    <div class="divider"> </div>
+                    <div class="fs"> See more Studio updates</div>
+                </div>
+                <!--End Column2 Second Box-->
+                <!--Start Column2 Third Box-->
+                <div class="secondBox">
+                    <div class="secondColumnTitle boxTitle">
+                    <div>Creator Insider</div>
+                    <div>
+                    <i class="fas fa-angle-left ffs op"></i>
```
Comment (james-highground) [inline yt_studio.html]: this indentation should be moved out

File: yt_studio_style.css [hunk 4]
```diff
@@ -2,7 +2,7 @@
     display: flex;
     flex-direction: row;
     justify-content: center;
-    background-color: #f9f9f9;
+    background-color:#E9ecef;
```
Comment (james-highground) [inline yt_studio_style.css]: hex code should be consistent. either everything lower case or everything upper case

File: yt_studio.html (hunk 1 above)
Comment (AbigailZaw) [reply yt_studio.html]: Yes.Sir.I will.

File: yt_studio.html (hunk 2 above)
Comment (AbigailZaw) [reply yt_studio.html]: Yp.Sir.

File: yt_studio.html (hunk 3 above)
Comment (AbigailZaw) [reply yt_studio.html]: Yes.Sam.

File: yt_studio_style.css (hunk 4 above)
Comment (AbigailZaw) [reply yt_studio_style.css]: Yes.Sir.

**Comment by james-highground** [comment]
let's do a zoom call and I can help you with indentation

**Pull request merged**